packages/dashboard/src/app/views/templateDetails/content/index.js

```diff
@@ -42,19 +41,6 @@ function Content({
   template,
   templateActions,
 }) {
-  const previousTemplateId = useRef(template?.id);
-
-  const {
-    actions: { scrollToTop },
-  } = useLayoutContext();
-
-  useEffect(() => {
-    if (template !== null && template?.id !== previousTemplateId.current) {
-      scrollToTop();
-      previousTemplateId.current = template.id;
-    }
-  }, [template, scrollToTop]);
```

Comment:
❤️ beautiful. good call on the slight behavior update here too to only scroll on grid click. nice touch.